src/debugpy/adapter/clients.py

```diff
@@ -12,6 +12,7 @@
 from debugpy.common import compat, fmt, json, log, messaging, sockets
 from debugpy.common.compat import unicode
 from debugpy.adapter import components, servers, sessions
+from debugpy.common.messaging import NO_RESPONSE
```

Comment:
We have a general policy for debugpy code to avoid imports of non-modules - they make things slightly worse with circular dependencies on 2.7, and break stuff that gets mutated by the module after it gets imported.

(`debugpy.common.compat` has a special dispensation for members that are intended to transparently override builtins, so it's okay to import `unicode`, `xrange` etc from it.)